### PR TITLE
update oc commands to build their own clients

### PR DESCRIPTION
**Pull request by deads2k**
In a factory free world, commands build their own clients.  This updates commands to build the clients they need based on config.  We are moving to a factory free world.

/assign @soltysh
/assign @juanvallejo

### CODE REVIEW

File: pkg/oc/cli/util/clientcmd/client.go [hunk 1]
```diff
@@ -1,193 +0,0 @@
-package clientcmd
```
Comment (soltysh) [inline pkg/oc/cli/util/clientcmd/client.go]: Die, die, die :tada: :heart: :+1:

**Comment by deads2k** [comment]
holding to give https://github.com/openshift/origin/pull/19327 priority

**Pull request merged**